cdap-ui/app/directives/dag-plus/my-dag.html

```diff
@@ -95,7 +95,7 @@
 
 <div class="my-js-dag"
     ng-class="{'disabled': DAGPlusPlusCtrl.isDisabled, 'normal-cursor': disableNodeClick, 'preview-mode': previewMode }"
-    ng-click="DAGPlusPlusCtrl.clearNodeSelection()">
+    ng-click="DAGPlusPlusCtrl.toggleNodeMenu(null); DAGPlusPlusCtrl.openDagMenu(false);">
```

Comment:
`DagPlusPlusCtrl.toggleNodeMenu` takes a `nodeName` param, which opens the hamburger menu for that node. Here we want to close any menu that's still open, so that's why we set the value of the param to be `null`